++ org/jfree/chart/renderer/category/AbstractCategoryItemRenderer.java
-- org/jfree/chart/renderer/category/AbstractCategoryItemRenderer.java
@@ -1796,7 +1796,7 @@
        CategoryDataset dataset = this.plot.getDataset(index);
+       if (dataset == null) {
-       if (dataset != null) {
            return result;
        }
        int seriesCount = dataset.getRowCount();
